types/nested-error-stacks/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for nested-error-stacks 2.1
+// Project: https://github.com/mdlavin/nested-error-stacks
+// Definitions by: Wouter van Heeswijk <https://github.com/woutervh->
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default class NestedError extends Error {
```

Comment:
It seems like the library not using `export default`, so this .d.ts file should not either.

Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html).